### ISSUE TITLE
Retain the display of function assignments above the collapse accordion on arrangement cards

### ISSUE THREAD

**Issue by LarsKemmann** [CareTogether/CareTogetherCMS#439]

**Comment by davidmalament** [comment]
This one confuses me...I'm not following what the end goal is. From what I can tell, this information is the _only_ thing inside the accordion, so if we move it outside, shouldn't we just get rid of the whole accordion?

**Comment by LarsKemmann** [comment]
The accordion contains both function assignments and activity requirements. The ask was to keep the function assignments visible so the various arrangements can be distinguished more easily (e.g., "hosting child X with family Y and coach Z").
________________________________

**Comment by davidmalament** [comment]
In my test environment this is all I see. Are these just function assignments and there are no activity requirements visible?

![Image](https://user-images.githubusercontent.com/5673847/172012187-83879bc6-460b-4293-b3e0-0e98b2a25a79.png)

**Comment by LarsKemmann** [comment]
In that screenshot, the Advocacy Agreement is an activity requirement. The others are functions (that haven't been assigned yet).
________________________________

**Comment by LarsKemmann** [comment]
@davidmalament I might have a better idea for how to approach this, that builds on the work I'm doing in #453. Let's discuss when we talk tomorrow.

**Comment by LarsKemmann** [comment]
So I don't forget... it would be good if the *assigned* people/families for an arrangement could *always* be shown. This can be in read-only plain text, below the partnering family's person's name. That would also align well with a potential card-and-side-sheet redesign concept -- key completed information remains visible on the summary cards, and all the rest is shown in side sheets (instead of accordions).

**Comment by Ad-Mortem** [comment]
Working on it

**Comment by Ad-Mortem** [comment]
Do we want the agreements moved up as well like this? 
![2](https://github.com/CareTogether/CareTogetherCMS/assets/132250085/49a61b40-0751-4a4d-a16d-40d01b5603a6)
Or do we want to keep them below like this? 
![1](https://github.com/CareTogether/CareTogetherCMS/assets/132250085/f99b7042-fdae-466a-8140-d5b0d74a9451)
I'm assuming moving them all up together.

**Comment by LarsKemmann** [comment]
No, at least for now, requirements (incomplete, exempted, and completed) should stay below so they can be collapsed with the accordion. So only the function assignments should be above the accordion.

Which, as I started to document in https://github.com/CareTogether/CareTogetherCMS/issues/439#issuecomment-1146967642, makes for some inconsistency. As a result, what I *really* think would make the most sense is to do what I described in that comment:

- keep all the requirements and function assignment rows as they are, below the accordion
- *add* a new (duplicative) display of just the actual function assignments (in the screenshots above, no assignments have been made of functions to any individuals or families), *above* the accordion

**Comment by LarsKemmann** [comment]
I forgot to add - an additional benefit of doing that would be that we can show the assigned host families etc. on the Partnering Families screen where only the "summary" versions of the arrangement cards are shown.

**Comment by AGNMM18** [comment]
Yes!  Please do this, and can we add searching by volunteer name as opposed to partnering family name as well?
